examples/child_window.rs

```diff
@@ -37,12 +38,14 @@ fn main() {
     let mut windows = HashMap::new();
 
     let event_loop: EventLoop<()> = EventLoop::new();
-    let parent_window = WindowBuilder::new()
-        .with_title("parent window")
-        .with_position(Position::Logical(LogicalPosition::new(0.0, 0.0)))
-        .with_inner_size(LogicalSize::new(640.0f32, 480.0f32))
-        .build(&event_loop)
-        .unwrap();
+    let parent_window = Rc::new(
```

Comment:
What's the reason we have `Rc` now all over the place for the builder?

Reply:
This is so the `softbuffer` contexts can hold a reference to the window/display objects. Essentially the new system prevents the issue we saw earlier where Wayland would crash if the `softbuffer` contexts outlived the event loop. The flip side of this is that you need to have a defined type that the `softbuffer` can hold on to (like an `&'a Window` or, in this case, an `Rc<Window>`) so that it can keep the window handle alive.

Reply:
Then if you're fixing this particular issue you should remove the `ManuallyDrop` otherwise I don't see what it tries to do so?